manifests/puppetdb.pp

```diff
@@ -140,10 +140,32 @@
       'url'  => 'puppetlabs.puppetdb.database:name=PDBWritePool.pool.Wait' },
   ]
 
+  $ha_sync_metrics = [
+    { 'name' => 'puppetdb_ha_last-sync-succeeded',
```

Comment:
@Sharpie can you remove `puppetdb_` from the front of all of these?  That'll make them match the other metrics.

Reply:
Done.